internal/npm_install/generate_build_file.js

```diff
@@ -675,6 +682,17 @@ function filterFiles(files, exts = []) {
   // Files with spaces (\x20) or unicode characters (<\x20 && >\x7E) are not allowed in
   // Bazel runfiles. See https://github.com/bazelbuild/bazel/issues/4327
   files = files.filter(f => !f.match(/[^\x21-\x7E]/));
+  files = files.filter(f => {
+    const basename = path.basename(f);
+    // Bazel files of non-nested node_modules would have been renamed earlier with a _ prefix
+    if (/^WORKSPACE$/i.test(basename) || /^BUILD$/i.test(basename) ||
```

Comment:
nit: Could potentially test the underscore prefix in the same regex

Reply:
👍